docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/channels-reference-guide.md

```diff
@@ -64,29 +64,30 @@ A channel can be connected to a DXP site. Alternately, a channel can be connecte
 
 ## Notification Queue
 
-Liferay Commerce can be configured to send email notifications that are triggered by a variety of store events. When an event triggers a notification (for example, an order is placed), the notification is logged in the channel's _Notification Queue_ tab. By default, the Liferay Commerce instance checks whether a notification has been sent every 15 minutes and deletes any unsent notifications after 43200 minutes (30 days).
+Liferay Commerce can be configured to send email notifications that are triggered by a variety of events. When an event triggers a notification (for example, an order is placed), the notification is logged in the channel's _Notification Queue_ tab. By default, the Liferay Commerce instance checks whether a notification has been sent every 15 minutes and deletes any unsent notifications after 43200 minutes (30 days).
 
 See the [Configuring the Commerce Notification Queue](../../../store-administration/sending-emails/configuring-the-commerce-notification-queue.md) article for more information.
 
 ## Notification Templates
 
 Notification Templates allow users to customize their email notifications and define the triggering events.
 
-To learn more about sending store emails notifications, see [Store emails](../../../store-administration/sending-emails/store-emails.md).
+To learn more about sending emails notifications, see [Store emails](../../../store-administration/sending-emails/store-emails.md).
 
 To learn more about creating and using notification templates, see [Using Notification Templates](../../../store-administration/sending-emails/using-notification-templates.md).
 
 ![Create notification templates.](./channels-reference-guide/images/11.png)
 
 ## Category Display Pages
 
-The Category Display Pages tab displays the list of all Category Display pages created on the store site. A _Category Display Page_ allow users to substitute a specific DXP site page that displays all products tagged with a certain Category. Users should first create [product categories](../products/organizing-your-catalog-with-product-categories.md), associate them with the desired products, and then create the pages.
```

Comment:
"tagged with a certain" can cause confused between categories and tags.

Reply:
Revised